src/Symfony/ResponseAssertions.php

```diff
@@ -20,4 +20,47 @@ protected static function assertJsonResponse(array $expected, JsonResponse $resp
         $content = json_decode($response->getContent(), true);
         Assert::assertSame($expected, $content);
     }
+
+    protected static function assertResponse(Response $response, int $expectedStatusCode, ?string $expectedMessage = null): void
+    {
+        self::assertStatusCode($response, $expectedStatusCode);
+
+        if ($expectedMessage !== null) {
+            self::assertResponseMessage($response, $expectedMessage);
+        }
+    }
+
+    protected static function assertResponseIsSuccessful(Response $response, ?string $expectedMessage = null): void
+    {
+        self::assertResponse($response, Response::HTTP_OK, $expectedMessage);
+    }
+
+    protected static function assertResponseIsRedirect(Response $response, ?string $expectedMessage = null): void
+    {
+        self::assertResponse($response, Response::HTTP_MOVED_PERMANENTLY, $expectedMessage);
```

Comment:
rename method to assertResponseIsRedirection, and use response->isRedirection instead of only allowing status 301.
instead of expectedMessage here, maybe assert the location url in the header?